### PR TITLE
Implement Desktop Version on my Portfolio Website

### PR DESCRIPTION
**Pull request by fmalgarini**
This pull request includes changes to make the Portfolio website compatible with desktop screens using media queries. The changes were made based on the selected Figma design template for the desktop version.

Key Changes:
- Implemented media queries to make the website responsive for desktop screens (>= 768px).
- Utilized Grid and Flexbox for element layout.
- Ensured the design details (font, colors, images, text, margins) match the Figma template.

General Requirements Met:
- No linter errors.
- Followed the correct Github flow.
- Documented the work in a professional way.

HTML/CSS Best Practices:
- Followed the best practices for HTML & CSS.

### CODE REVIEW

**Comment by fmalgarini** [comment]
Hello reviewer, it's a pleasure. 👋👋
 I can't find a way to place my background-image, and make it fixed. In the css it is located in the ".presentation-container" class. Would you be so kind as to guide me on how to do it?

gif

File: index.html [hunk 1]
```diff
@@ -4,44 +4,47 @@
...
-    <nav>
+    <nav class="background-image">
       <ul class="nav-container">
-        <li class="nav-items"><a href="#"></a>MyLogo</li>
-        <li class="nav-items assetNav"><a href="#"></a></li>
+        <a href="#" class="nav-items">Facundo Malgarini</a>
+        <a href="#" class="nav-items assetNav"></a>
+        <a href="#" class="nav-items nav-links navPortfolio">Porfolio</a>
+        <a href="#" class="nav-items nav-links navAbout">About</a>
+        <a href="#" class="nav-items nav-links navContact">Contact</a>
```
Comment (usorfaitheloho) [inline index.html]: Hello @fmalgarini, you did a great job with your portfolio, well-done 👍 , kindly adjust your navigation items such that it is aligned to the right while the `logo` is aligned to the left just as it is in the figma design given 😄
Also, you should reduce the size of your background images and place them in the right position away from the text as seen in the template given 👍  

Figma design :
image

Your design :
image

File: style.css [hunk 2]
```diff
@@ -509,3 +522,304 @@ textarea::placeholder {
...
+    width: 45%;
+  }
+
+  .aboutMyself-container p {
+    padding: 0;
+    margin: 0;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 30px;
+  }
```
Comment (usorfaitheloho) [inline style.css:795]: kindly align the technologies listed in the `about me ` section to the left as shown the Figma design given to have a better design

Figma design:
image

Your design :
image

File: style.css [hunk 3]
```diff
@@ -509,3 +522,304 @@ textarea::placeholder {
...
+    padding-top: 0.5rem;
+    width: 3rem;
+  }
+
+  #css {
+    padding: 0;
+    padding-top: 0.5rem;
+    width: 2rem;
+  }
+}
```
Comment (usorfaitheloho) [inline style.css:891]: You seem to not be having a proper view of the background images in the contact section, could you please add all the images in the section and also remember to place them in the right position similar to what we have in our figma template 😄 

You should also implement the following changes for your contact section 👍 

1. The text after the section title is aligned in three lines as seen in the Figma template, could you please implement this in your project
2. Please I think you should increase the width of your contact-form input to be similar to the design and also use the right font styling given in our design
3. kindly align the `Get in touch` button to the center and use the font-styling given in the template 

Figma template :
image

Your design :
image

**Review by usorfaitheloho — CHANGES_REQUESTED** [review]
> Hi @fmalgarini  ,
> 
> Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!

  [ ](https://camo.githubusercontent.com/679757134a8fee76cfe1941f9f3ecb0e0cf56dd02da4498268ce31f0dacdfca3/68747470733a2f2f6d65646961302e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966)
> 
> ## Highlights
> * Good UI ✔️
> * Descriptive pull request ✔️
> * Used GitHub flow ✔️
> 
> ## Required Changes ♻️
> _Check the comments under the review._
> 
> ## Optional suggestions
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 4]
```diff
@@ -154,30 +167,33 @@ <h5>LET'S CONNECT</h5>
...
   </div>
   <div class="languagesContainer">
-    <ul>
-      <li class="languages"><img src="assets/javascriptAsset.svg" alt="jsicon"> <span id="js">JavaScript</span></li>
-      <li class="languages"><img src="assets/htmlAsset.svg" alt="htmlicon"><span id="html">HTML</span></li>
-      <li class="languages"><img src="assets/cssAsset.svg" alt="cssicon"><span id="css">CSS</span></li>
+    <ul class="container-desktop">
+      <li class="languages"><img class="languages-img" src="assets/javascriptAsset.svg" alt="jsicon"> <span id="js">JavaScript</span></li>
+      <li class="languages"><img class="languages-img" src="assets/htmlAsset.svg" alt="htmlicon"><span id="html">HTML</span></li>
+      <li class="languages"><img class="languages-img" src="assets/cssAsset.svg" alt="cssicon"><span id="css">CSS</span></li>
```
Comment (divinecharlotte) [inline index.html:182]: - [ ] Kindly as requested by the previous reviewer consider aligning the technologies names  listed in the About me  section to the left as shown in the Figma design given to make your design an exact duplicate of the given one 
- your design 👇 
Screen Shot 2023-06-17 at 20 59 05

- Figma design 👇 
Screen Shot 2023-06-17 at 21 00 30

File: index.html [hunk 5]
```diff
@@ -154,30 +167,33 @@ <h5>LET'S CONNECT</h5>
...
   </div>
-  <hr>
+  <hr> 
+</div> 
 </section>
+
 <!--  ______________________________Footer-FORM-section___________________________  -->
-<footer>
+<footer class="footer-mobile footer-desktop">
   <section class="form-container">
```
Comment (divinecharlotte) [inline index.html:200]: - [ ] For this section as requested by the previous reviewer kindly consider sticking to the Figma design as much as possible by adding the exact background image 
- your design 👇 
Screen Shot 2023-06-17 at 21 05 01

- Figma design 👇 
Screen Shot 2023-06-17 at 21 07 59

**Review by divinecharlotte — CHANGES_REQUESTED** [review]
Hi @fmalgarini    , 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

## To Highlight 💯 

- Use of GitHub flow👍 
- Descriptive pr 👍 

 




 

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by KanzaTahreem — APPROVED** [review]
Linter errors were previously fixed however, I forgot to refresh the page 😓

**Review by KanzaTahreem — APPROVED** [review]
# Approved 🙌

Hi @fmalgarini 

Great work nailing it on the first attempt 💪 **_Your project is COMPLETE! There is nothing else to say other than... it's time to merge it! 🚀 Congratulations!_**



## Highlights 🎯
There are some excellent points that you have made perfectly. Keep up the great work! 👏

-  ✅ Styling is looking great, perfect!
-  ✅ Great work using the right GitHub Flow 
-  ✅ Your README looks good, well done!
-  ✅ Linters are working properly, great job! 

## Optional suggestions ❕
N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**